public/app/features/storage/FolderView.tsx

```diff
@@ -0,0 +1,87 @@
+import { css } from '@emotion/css';
+import React from 'react';
+import AutoSizer from 'react-virtualized-auto-sizer';
+
+import { DataFrame, GrafanaTheme2 } from '@grafana/data';
+import { Table, useStyles2 } from '@grafana/ui';
+
+import { UploadView } from './UploadView';
+import { StorageView } from './types';
+
+interface Props {
+  listing: DataFrame;
+  path: string;
+  onPathChange: (p: string, view?: StorageView) => void;
+  view: StorageView;
+}
+
+export function FolderView({ listing, path, onPathChange, view }: Props) {
+  const styles = useStyles2(getStyles);
+
+  switch (view) {
+    case StorageView.Config:
+      return <div>CONFIGURE?</div>;
```

Comment:
Currently this is only shown for the top level -- so it would be the storage configuration page.  If we need to support deeper symlink style things, it could be shown on deeper levels